src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -558,7 +539,7 @@ GLHEVertProps::GLHEVertProps(EnergyPlusData &state, std::string const &objName,
 std::shared_ptr<GLHEVertProps> GetVertProps(EnergyPlusData &state, std::string const &objectName)
 {
     // Check if this instance of this model has already been retrieved
-    for (auto &thisProp : state.dataGroundHeatExchanger->vertPropsVector) {
+    for (auto const &thisProp : state.dataGroundHeatExchanger->vertPropsVector) {
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:561]:(style),[constVariable],Variable 'thisProp' can be declared as reference to const